### DISCUSSION TITLE
lvim's current directory to always be where it was launched from

### DISCUSSION THREAD

**Discussion by cassepipe** [LunarVim/LunarVim#4085] [Q&A] [answered]
No matter where the directory I launch lvim from, if it it finds there's a git repo above it will place itself at the root.
(I am checking with `:pwd`)

It's a good default but I'd personally like to avoid that behavior. How can I do that ?

**Comment by LostInTheLogs** [comment, marked as answer]
disable project.nvim

**Reply by cassepipe** [reply]
Thanks ! 
For the record, in 1.3, in order to no disable project.nvim and also remove it from the welcome screen I added to my config :
```lua
-- Disbable project.nvim
lvim.builtin.project.active = false
-- Remove corresponding entry in dashboard
table.remove(lvim.builtin.alpha.dashboard.section.buttons.entries, 3)
```

**Reply by singh-gyan** [reply]
That's what I was looking for. Thanks a lot!!

**Reply by cassepipe** [reply]
@singh-gyan 
Note I have since rolled that back since the capacity of the linters to work well seem to depend on the ability to identify a root directory for your project.
